src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -125,33 +125,79 @@
         [:div {:style {:clear "both"}}]])}))
 
 
+(defn- modal-background [state]
+  {:backgroundColor "rgba(82, 129, 197, 0.4)"
+   :display (if (:overlay-shown? @state) "block" "none")
+   :overflowX "hidden" :overflowY "scroll"
+   :position "fixed" :zIndex 9999
+   :top 0 :right 0 :bottom 0 :left 0
+   :textAlign "left"})
+
+(def modal-content
+  {:transform "translate(-50%, 0px)"
+   :backgroundColor "#F4F4F4"
```

Comment:
Nit: I've been using lowercase for these because I think they are easier to read and type. I don't have a strong preference, just want us to be consistent.